cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -932,6 +932,33 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
           }
           {{- end}}
 
+          {{- range .DependsOn}}
```

Comment:
When using `Eureka` and `Config Server`, inject the following Env Vars.
following the doc: 
https://learn.microsoft.com/en-us/azure/container-apps/java-eureka-server?tabs=azure-cli#bind-your-container-app-to-the-eureka-server-for-spring-java-component
https://learn.microsoft.com/en-us/azure/container-apps/java-config-server?tabs=azure-cli#bind-your-container-app-to-the-config-server-for-spring-java-component